src/applications/static-pages/download-1095b/components/App/utils.js

```diff
@@ -51,3 +51,29 @@ export const notFoundComponent = () => {
     </va-alert>
   );
 };
+
+export const unavailableComponent = () => {
+  return (
+    <va-alert
+      close-btn-aria-label="Close notification"
+      status="warning"
+      visible
+    >
+      <h3 slot="headline">1095-B download unavailable at this time</h3>
+      <div>
+        <p>
+          Please check back later or if you need immediate assistance with this
+          tax form, call the Enrollment Center at{' '}
+          <a href="tel:+18772228387" aria-label="1 8 7 7 2 2 2 8 3 8 7">
+            1-877-222-8387
+          </a>{' '}
```

Comment:
It would be better to use the [telephone web component](https://design.va.gov/components/telephone) here.

Reply:
updated these to use Telephone component now. Took the liberty to update a couple other spots in our 1095B widget code to use the Telephone component as well